airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -106,6 +106,7 @@ definitions:
       - emitted_at
     properties:
       type:
+        title: "trace type"
```

Comment:
```suggestion
        title: "trace type" # this title is required to avoid python codegen conflicts with the "type" parameter in AirbyteMessage. See https://github.com/airbytehq/airbyte/pull/12581
```